spec/fortnox/api/repositories/examples/save_examples.rb

```diff
@@ -0,0 +1,41 @@
+shared_examples_for '.save' do |attribute, fortnox_attribute|
```

Comment:
As a further comment on this I think you should only take the `attribute_hash_key_name` as an argument since the repository in question already has functions that map from a JSON key name to a hash key name, use them instead :)